### ISSUE TITLE
Functional Groups没有label.json文件

### ISSUE THREAD

**Issue by luow23** [X1nyuLu/PACE#4]
作者，您好，请问为什么Functional Groups没有label.json文件？而且训练时代码报错有多个label。

**Comment by X1nyuLu** [comment]
感谢您的关注。官能团识别是一个多标签分类任务，即一条光谱所对应的分子可能包含多个官能团。即输入一条光谱，需要预测多个官能团的存在与否，而非只需将其划为某一个具体的类别。因此无法像其他数据集一样给出精确的`label.json`。此外，训练时需要使用`torch.nn.BCELoss()`而非`torch.nn.CrossEntropyLoss()`进行优化，您可以查看一下是否正确地使用了损失函数。

**Comment by luow23** [comment]
好的，谢谢！
